### ISSUE TITLE
Publish new services

### ISSUE THREAD

**Issue by daniel-abramov** [dylanmckay/mdns#10]
Maybe I am missing something, but how do I publish a new service? It seems that currently only the discovery is supported, right? If so, are there any plans to add the support for publishing the new services? (like the one can do with Bonjour / Avahi)

**Comment by dylanmckay** [comment]
> Maybe I am missing something, but how do I publish a new service? It seems that currently only the discovery is supported, right?

Yup - at the moment, this library is just a mDNS client. There's no particular reason for this - I simply haven't had a need for an mDNS server. Patches welcome!

**Comment by superdave** [comment]
Hm, I might be interested in this, looks like there is already a listener class that mainly needs some request handling added...
